src/platform/silabs/rs911x/BLEManagerImpl.cpp

```diff
@@ -59,13 +59,14 @@ extern "C" {
 
 // Used to send the Indication Confirmation
 uint8_t dev_address[RSI_DEV_ADDR_LEN];
-uint16_t ble_measurement_hndl;
+uint16_t rsi_ble_measurement_hndl;
+uint16_t rsi_ble_gatt_server_client_config_hndl;
```

Comment:
These variables are acting as global variables since they are not in an anounymous namespace. This applies to most if not all the variables in this file.

A small tip for anonymous namespaces, it replaces the need for the `static` keyword in the cpp files.

Reply:
done